distribution/pom.xml

```diff
@@ -83,6 +74,7 @@
 				<executions>
 					<execution>
 						<id>attach-javadoc</id>
+						<phase>install</phase>
```

Comment:
I'm not sure if this forces the artifact to be built or not... MVN is  This is a job for **-X**